coreTable/CoreTableUtils.ts

```diff
@@ -360,14 +362,29 @@ export const appendRowsToColumnStore = (
     return newColumnStore
 }
 
+export const getColumnStoreLength = (store: CoreColumnStore) => {
+    for (const slug in store) {
+        return store[slug].length
+    }
+    return 0
+}
+
 export const concatColumnStores = (stores: CoreColumnStore[]) => {
+    if (!stores.length) return {}
+
+    const lengths = stores.map(getColumnStoreLength)
+    const slugs = Object.keys(first(stores)!)
```

Comment:
Ah wait, forget about my comment above. What if this was a param?

```
export const concatColumnStores = (stores: CoreColumnStore[], slugsToKeep = Object.keys(first(stores)!)) => {
```

Then it would be clear that only certain slugs will be kept and theoretically you could specify to keep them all

Reply:
Great idea!